### PR TITLE
This prevents people from being a idiot.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by AliiTea** [comment]
10/10 description on pr

File: .gitignore [hunk 1]
```diff
@@ -9,4 +9,5 @@
 *.lk
 *.int
 *.backup
-*.txt
\ No newline at end of file
+*.txt
+*.int
```
Comment (nalarmg) [inline .gitignore:13]: Duplicate *.int?

**Comment by danhibiki** [comment]
Looks like it would be mostly helpful

**Comment by nalarmg** [comment]
Duplicated *.int, no sign of response/addressing. Closing.

**Pull request closed without merging**